ban/core/resource.py

```diff
@@ -126,25 +128,54 @@ def as_version(self):
         """Resources plus relations references and metadata."""
         return self.serialize({f: {} for f in self.versioned_fields})
 
+    @property
+    def status(self):
+        return 'deleted' if self.deleted_at else 'active'
+
+    @classmethod
+    def active(cls):
```

Comment:
It might be safer to override `select` here and provide a custom way of returning results including deleted ones.